.github/workflows/ci.yml

```diff
@@ -73,11 +73,28 @@ jobs:
       redis:
         image: redis:${{ matrix.redis }}
         ports:
-        - 6379:6379
+        - 7000:7000
         options: --entrypoint redis-server
 
     steps:
     - uses: actions/checkout@v2
+    - name: Test redis cluster
```

Comment:
I think we should add cluster tests separately from the main tests as `redis-cluster-test`.